docs_outdoornav_user_manual/features/mission_scheduler.mdx

```diff
@@ -5,3 +5,40 @@ sidebar_position: 4
 toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
+
+### Scheduling missions
+
+By selecting the "SCHEDULER" option from the upper left hand hamburger menu, the user can open the Mission scheduler interface. This interface 
+allows users to schedule groups of missions to run at a later time. These schedules can be set to run only once or to be looped over a period of time.
+For example, a user could create a schedule that will run 3 missions starting at 9 AM and run every hour 8 times. Once the last iteration is run, the 
+schedule will disable itself and can be re-enabled to run again later.  The view itself can be found in this screen shot and is elaborated further afterwards.
+
+<center>
+  <figure>
+    <img src="/img/outdoornav_images/scheduler_screen.png" />
+    <figcaption>Mission Scheduler View</figcaption>
+  </figure>
+</center>
+
+#### Adding/Updating a Schedule
+
+The user can either create a new schedule by filling in the appropriate fields or they may edit a schedule by selecting the pencil icon for that schedule 
+in the bottom table.  The input fields are outlined as follows:
+
+-   <strong>Name</strong>: The name of the schedule.
+-   <strong>Start Time</strong>: The local time that the schedule will start. This is stored as UTC time on the robot.

```

Comment:
Whose local time? The local time on the laptop that has the browser open? Or the local time on the robot?